region/brinstar/green/Brinstar Reserve Tank Room.json

```diff
@@ -232,35 +237,41 @@
     {
       "link": [2, 1],
       "name": "Base",
-      "requires": []
+      "requires": [],
+      "flashSuitChecked": true
     },
     {
       "link": [2, 3],
       "name": "Base",
       "requires": [
         "Morph"
-      ]
+      ],
+      "flashSuitChecked": true
     },
     {
       "link": [3, 2],
       "name": "Base",
       "requires": [
         "Morph"
-      ]
+      ],
+      "flashSuitChecked": true
     },
     {
       "link": [3, 2],
       "name": "G-mode Morph Remote Acquire Return",
       "requires": [
+        "canEnterGMode",
```

Comment:
I was thinking of adding this tech to all g-mode strats that don't use a door node, so they no longer show in `Basic`. I think it could help make things clearer, but maybe the tech would need to be renamed as well?

Reply:
That makes sense.
The tech name still sounds like it fits, to me.